cp -v is not generally portable

<!-- Hi! Thanks for filing an issue. It will be read with care by human beings. Can we ask you to please fill out this template and not simply demand new features or send in complaints? Thanks! -->
<!-- Also please search the existing issues (both open and closed) to see if your report might be duplicate -->
- [X] This is not a support question, I have read [about opensource](https://www.powerdns.com/opensource.html) and will send support questions to the IRC channel, [GitHub Discussions](https://github.com/PowerDNS/pdns/discussions/) or the mailing list.
- [X] I have read and understood the ['out in the open' support policy](https://blog.powerdns.com/2016/01/18/open-source-support-out-in-the-open/)
- [X] I have read and understood the [PowerDNS AI policy](https://github.com/PowerDNS/pdns/blob/master/AI_POLICY.md)

<!-- Tell us what is issue is about -->
 - Program: Recursor
 - Issue type: Bug report

### Short description
cp -pv is not generally portable and we should avoid using it in Makefile.am and Makefile.in.
It's works on Linux/MacOS some BSD, not not all POSIX strict systems.
The problematic is "-v" param.

### Environment
 - Operating system: NetBSD/pkgsrc.org/SmartOS
 - Software version: 5.4.1
 - Software source: compiled yourself

### Steps to reproduce

List where it's hardcoded.

```
./pdns-recursor-5.4.1/rec-rust-lib/rust/Makefile.am:25: cp -pv target/$(RUSTC_TARGET_ARCH)/release/librecrust.a librecrust.a
./pdns-recursor-5.4.1/rec-rust-lib/rust/Makefile.am:26: cp -pv target/$(RUSTC_TARGET_ARCH)/cxxbridge/recrust/src/lib.rs.h lib.rs.h
./pdns-recursor-5.4.1/rec-rust-lib/rust/Makefile.am:27: cp -pv target/$(RUSTC_TARGET_ARCH)/cxxbridge/recrust/src/web.rs.h web.rs.h
./pdns-recursor-5.4.1/rec-rust-lib/rust/Makefile.am:28: cp -pv target/$(RUSTC_TARGET_ARCH)/cxxbridge/recrust/src/misc.rs.h misc.rs.h
./pdns-recursor-5.4.1/rec-rust-lib/rust/Makefile.am:29: cp -pv target/$(RUSTC_TARGET_ARCH)/cxxbridge/rust/cxx.h cxx.h
./pdns-recursor-5.4.1/rec-rust-lib/rust/Makefile.in:592:        cp -pv target/$(RUSTC_TARGET_ARCH)/release/librecrust.a librecrust.a
./pdns-recursor-5.4.1/rec-rust-lib/rust/Makefile.in:593:        cp -pv target/$(RUSTC_TARGET_ARCH)/cxxbridge/recrust/src/lib.rs.h lib.rs.h
./pdns-recursor-5.4.1/rec-rust-lib/rust/Makefile.in:594:        cp -pv target/$(RUSTC_TARGET_ARCH)/cxxbridge/recrust/src/web.rs.h web.rs.h
./pdns-recursor-5.4.1/rec-rust-lib/rust/Makefile.in:595:        cp -pv target/$(RUSTC_TARGET_ARCH)/cxxbridge/recrust/src/misc.rs.h misc.rs.h
./pdns-recursor-5.4.1/rec-rust-lib/rust/Makefile.in:596:        cp -pv target/$(RUSTC_TARGET_ARCH)/cxxbridge/rust/cxx.h cxx.h
```

### Expected behaviour
Remove -v

### Actual behaviour

SmartOS/illumos

[...]
```
   Compiling base64 v0.22.1
    Finished `release` profile [optimized] target(s) in 2m 46s cp -pv target//release/librecrust.a librecrust.a
cp: illegal option -- v
Usage: cp [-a] [-f] [-i] [-p] [-@] [-/] f1 f2
       cp [-a] [-f] [-i] [-p] [-@] [-/] f1 ... fn d1
       cp [-r|-R [-H|-L|-P]] [-a] [-f] [-i] [-p] [-@] [-/] d1 ... dn-1 dn
gmake[2]: *** [Makefile:592: librecrust.a] Error 2 
```
[...]

### Other information
Minor change, should be tested for impact, but once corrected we can skip to use patches to compile powerdns-recursor.

--- a/pdns/recursordist/rec-rust-lib/rust/Makefile.am
+++ b/pdns/recursordist/rec-rust-lib/rust/Makefile.am
@@ -22,11 +22,11 @@ librecrust.a lib.rs.h web.rs.h misc.rs.h: \
 	src/misc.rs \
 	src/web.rs
 	SYSCONFDIR=$(sysconfdir) NODCACHEDIRNOD=$(localstatedir)/nod NODCACHEDIRUDR=$(localstatedir)/udr $(CARGO) build --release $(RUST_TARGET) --target-dir=$(builddir)/target --manifest-path ${srcdir}/Cargo.toml
-	cp -pv target/$(RUSTC_TARGET_ARCH)/release/librecrust.a librecrust.a
-	cp -pv target/$(RUSTC_TARGET_ARCH)/cxxbridge/recrust/src/lib.rs.h lib.rs.h
-	cp -pv target/$(RUSTC_TARGET_ARCH)/cxxbridge/recrust/src/web.rs.h web.rs.h
-	cp -pv target/$(RUSTC_TARGET_ARCH)/cxxbridge/recrust/src/misc.rs.h misc.rs.h
-	cp -pv target/$(RUSTC_TARGET_ARCH)/cxxbridge/rust/cxx.h cxx.h
+	cp -p target/$(RUSTC_TARGET_ARCH)/release/librecrust.a librecrust.a
+	cp -p target/$(RUSTC_TARGET_ARCH)/cxxbridge/recrust/src/lib.rs.h lib.rs.h
+	cp -p target/$(RUSTC_TARGET_ARCH)/cxxbridge/recrust/src/web.rs.h web.rs.h
+	cp -p target/$(RUSTC_TARGET_ARCH)/cxxbridge/recrust/src/misc.rs.h misc.rs.h
+	cp -p target/$(RUSTC_TARGET_ARCH)/cxxbridge/rust/cxx.h cxx.h
 
 clean-local:
 	rm -rf librecrust.a src/lib.rs lib.rs.h web.rs.h cxx.h misc.rs.h target

